### DISCUSSION TITLE
Help needed backmapping CG to All-Atom

### DISCUSSION THREAD

**Discussion by aashuph16221** [Martini-Force-Field-Initiative/MartiniDiscussions#14] [Tools]
Hello,

I have run the coarse-grained (CG) dynamics and now I want to convert or backmap it to the all-atom representation. However, I am having difficulty with the conversion process.

The command I used is:
python2 backward.py -f resid.pdb  -o aa.gro  -from martini3 -to charmm36 -p topol.top

rtini_to_aa/new_test/1.test/test_run/Mapping/ile.charmm36.map).
Traceback (most recent call last):
  File "backward.py", line 651, in <module>
    struc = Structure(options["-f"].value,strict=options["-strict"].value)
  File "backward.py", line 417, in __init__
    d12.append(d12[-1])
IndexError: list index out of range

I have checked the file multiple times but still encounter the same issue. Could you please help me resolve this error?

Regards,

**Comment by JingkaiZeng** [comment]
Same issue here, do someone find the solution yet?

Thanks in advance

**Reply by aashuph16221** [reply]
I used for protein conversion = https://mstool.readthedocs.io/en/latest/#
